code.js

```diff
@@ -198,7 +198,7 @@ function listPlugins(ul) {
 		$u.element.create('li', {
 			contents: {
 				tag: 'a',
-				prop: {
+				properties: {
```

Comment:
Nice catch!

Reply:
The `prop` alias is&nbsp;only available as&nbsp;a&nbsp;fallback when passed to&nbsp;the&nbsp;`$u.element.set(…)`&nbsp;method (it’s&nbsp;ignored&nbsp;in&nbsp;`$u.element.create(…)`, because `create(…)` always creates a&nbsp;`properties`&nbsp;key).